k8s/src/charm.py

```diff
@@ -236,6 +246,14 @@ def __init__(self, *args: Any):
             self.vault_kv.on.vault_kv_client_detached, self._on_vault_kv_client_detached
         )
 
+    def _resource_reqs_from_config(self) -> ResourceRequirements:
+        limits = {
+            "cpu": self.model.config.get("cpu"),
+            "memory": self.model.config.get("memory"),
+        }
+        requests = {"cpu": "750m", "memory": "1.5Gi"}
```

Comment:
In `charmcraft.yaml`, we claim that requests are "automatically deduced", but it looks like we hardcode values here?